Assets/MRTK/Examples/Demos/EyeTracking/General/Scripts/TargetBehaviors/OnLoadStartScene.cs

```diff
@@ -6,6 +6,12 @@
 
 namespace Microsoft.MixedReality.Toolkit.Examples.Demos.EyeTracking
 {
+    public enum LoadOptions
```

Comment:
Can this be private, since it only seems to be used in this script?

```suggestion
    private enum LoadOptions
```